projects/exui-common-lib/src/lib/components/write-address/write-address.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="govuk-form-group">
+
+  <div *ngIf="!shouldShowDetailFields()">
+    <div class="govuk-form-group bottom-30 postcodeLookup" id="addressLookup"
```

Comment:
I can't find a styling for bottom-30. The GDS styling https://design-system.service.gov.uk/styles/spacing/ can be used instead.

Reply:
again, follows from toolkit component, will remove and see if I need to reset any spacing